### PR TITLE
Reminder translation

### PR DESCRIPTION
**Pull request by amma35**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

Add translation for reminders

Rebase for https://github.com/glpi-project/glpi/pull/3926

### CODE REVIEW

File: inc/remindertranslation.class.php [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+         }
+         return self::createTabEntry(self::getTypeName(Session::getPluralNumber()), $nb);
+      }
+
+      return '';
+   }
+
+
+   /**
+    * @param $item            CommonGLPI object
```
Comment (trasher) [inline inc/remindertranslation.class.php]: Doc is not OK, see https://glpi-developer-documentation.readthedocs.io/en/master/codingstandards.html#parameters-documentation

Comment (trasher) [reply inc/remindertranslation.class.php]: (on all methods)

File: inc/remindertranslation.class.php [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+         echo "};";
+         echo "</script>\n";
+
+         echo "<div class='center'>".
+              "<a class='vsubmit' href='javascript:addTranslation".$item->getID()."$rand();'>".
+              __('Add a new translation')."</a></div><br>";
+      }
+
+      $obj   = new self;
+      $found = $obj->find("`reminders_id`='".$item->getID()."'", "`language` ASC");
```
Comment (trasher) [inline inc/remindertranslation.class.php]: This is deprecated, you have to pass an array

File: inc/remindertranslation.class.php [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+            if ($canedit) {
+               echo "<td class='center'>";
+               Html::showMassiveActionCheckBox(__CLASS__, $data["id"]);
+               echo "</td>";
+            }
+            echo "<td>";
+            echo Dropdown::getLanguageName($data['language']);
+            echo "</td><td>";
+            if ($canedit) {
+               echo "<a href=\"" . $CFG_GLPI["root_doc"] . "/front/remindertranslation.form.php?id=".$data["id"] . "\">{$data['name']}</a>";
```
Comment (trasher) [inline inc/remindertranslation.class.php]: Use getFormURLWithID (or almost... :D )

**Comment by cedric-anne** [comment]
Hi,

Do you still want this to be merged ?
It should be rebase on 9.5/bugfixes branch as this is a new feature.

Regards

**Comment by tsmr** [comment]
Yes but i have no time to do this now :(

**Comment by tsmr** [comment]
New Pull request for 9.5
https://github.com/glpi-project/glpi/pull/7179

**Pull request closed without merging**